fastlane/jetpack_metadata/default/release_notes.txt

```diff
@@ -1,16 +1,8 @@
-We’ve added a new chat-based support channel with a specially trained bot that provides answers to all your app-related questions. That’ll do, bot. That’ll do. (Still want to talk to a human? As always, you can create a support ticket and talk to a Happiness Engineer.)
+* [***] Lock Screen Widgets are now available on Jetpack. Quickly view your site stats without unlocking your phone. [#21535]
+* [*] Block editor: Fix the obscurred "Insert from URL" input for media blocks when using a device in landscape orientation. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/6143]
+* [**] Block editor: Updated placeholder text colors for block-based themes [https://github.com/wordpress-mobile/gutenberg-mobile/pull/6182]
+* [**] Update the main site navigation on the My Site screen, making it easier to access the common site sections. The new shortcuts can be modified using the Personalize Home Tab screen. [21502](https://github.com/wordpress-mobile/WordPress-iOS/pull/21502)
+* [**] [internal] Refactor fetching one single media item. [#21567]
+* [**] [internal] Refactored how the total number of WordPress Media is fetched. [#21568]
+* [**] [internal] Refactor fetching metadata of VideoPress videos. [#21569]
```

Comment:
I was confused at first that this file was still containing the bullet-point release notes instead of the editorialized copy (especially since the `.po` file have been updated with said editorialized copy). But then I remembered about [this](https://github.com/wordpress-mobile/WordPress-iOS/blob/trunk/fastlane/lanes/localization.rb#L305-L310) which explains that inconsistency 😅